src/styles/globals.css

```diff
@@ -1,9 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Mukta:wght@500&display=swap');
```

Comment:
Fine for now to add the font like this but would be better to use something like [fontsource](https://fontsource.org/) ([see orcasite](https://github.com/orcasound/orcasite/blob/master/ui/src/styles/theme.ts) for example of how to use)